mmgen/models/architectures/ddpm/denoising.py

```diff
@@ -387,7 +387,8 @@ def forward(self, x_t, t, label=None, return_noise=False):
         if return_noise:
             output_dict['x_t'] = x_t
             output_dict['t_rescaled'] = t
-            output_dict['label'] = label
+            if self.num_classes > 0:
```

Comment:
Is this attribute ever be used? Maybe you can remove it, or at least add a test case for it.

Reply:
Unit test is added.